src/index.tsx

```diff
@@ -143,6 +149,17 @@ export default function BeforeAfterSlider({
             ? new IntersectionObserver(observerCallback, observerOptions)
             : null
     );
+
+    useEffect(() => {
+        if(typeof window !== "undefined"){
+            if(window.innerWidth < 769){
```

Comment:
This is not the best way to define a touch interface.
You can use this for it:
https://stackoverflow.com/questions/4817029/whats-the-best-way-to-detect-a-touch-screen-device-using-javascript

Reply:
oh!! thank you so much.